dashboard/app/assets/stylesheets/errors.scss

```diff
@@ -86,3 +86,17 @@
     max-width: 300px;
   }
 }
+
+.deprecated-course-page {
+  margin-bottom: 100px;
+
+  .deprecated-course-child {
+    display: inline-block;
+    max-width: 500px;
+  }
+
+  h1 {
+    color: $neutral_dark;
```

Comment:
I would also add   `font-family: $barlowSemiCondensed-semibold;` here, so it'll use the same fontFamily that our designSystem uses

Reply:
![Uploading image.png…]()

Reply:
<img width="1675" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/22244040/3a856713-ccad-4c34-8145-ee39c9f650bc">

Reply:
Isn't that font just used for headings, rather than multi-sentence paragraphs?

Reply:
Yes it is. But it's `h1`, not `p` tag currently

Reply:
If it's supposed to look like `p`, not `h1` then I'm ok with it

Reply:
Yes, I think the styling issue is that we want a big font, but not necessarily a heading.  Do the design guidelines have a recommendation for that?

Reply:
We agreed that we can have any semantic tag look like any 'visualAppearance'.
Meaning that `h1` can look like `bodyOneText` and vice versa.
We can stick with this version, or you can copy paste the styles from `visualAppearance` in `typography.module.scss` you want to use for this text

Reply:
I switched this to a `p` tag to be more semantically appropriate, with "big font" styling.